vnc.html

```diff
@@ -152,7 +152,7 @@ <h1 class="noVNC_logo" translate="no"><span>no</span><br />VNC</h1>
             <div class="noVNC_vcenter">
             <div id="noVNC_power" class="noVNC_panel">
                 <div class="noVNC_heading">
-                    <img src="app/images/power.svg"> Power
+                    <img alt="" src="app/images/power.svg"> Power
```

Comment:
That's just silencing the error. We should have a proper text in there. :)

Reply:
I should probably have given a better explanation, but in most of these cases the images are purely decorative icons that doesn't provide any added information. According to the HTML specification, an empty alt is the proper way to go here:

https://www.w3.org/TR/html51/semantics-embedded-content.html#icon-images